cmd/tap/db_helpers.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/bluesky-social/indigo/cmd/tap/models"
+	"gorm.io/gorm"
+)
+
+func deleteRepo(tx *gorm.DB, did string) error {
+	if err := tx.Delete(&models.RepoRecord{}, "did = ?", did).Error; err != nil {
+		return err
+	}
+	if err := tx.Delete(&models.ResyncBuffer{}, "did = ?", did).Error; err != nil {
+		return err
+	}
+	return tx.Delete(&models.Repo{}, "did = ?", did).Error
```

Comment:
will the account just get re-initialized if it is seen again? I kind of assume this should result in the status getting set to `deleted`, as a sort of tombstone row.